client/main.js

```diff
@@ -32,10 +33,10 @@ window.addEventListener( 'beforeunload', ( event ) => {
 const rootEl = document.getElementById( 'wc-connect-admin-container' );
 
 let render = () => {
-	const WCCShippingSettings = require( 'views/shipping' );
+	const RootView = 'shipping-label' === rootView ? require( 'views/shipping-label' ) : require( 'views/shipping' );
```

Comment:
In this case, I think having a conditional `require` is a reasonable compromise. If we `require` the 2 modules and then decide which one to render, then we are pulling the entire dependency trees of both the components in every case. This may count as micro-optimization, but if it makes start-up 50ms quicker I think is worth having one "weird" line.